src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -1,14 +1,34 @@
 package racingcar.domain
 
-class Car {
+data class Car(val name: String) {
 
-    val history = mutableListOf<CarAction>()
+    init {
+        require(name.length <= 5)
+    }
+
+    private val _history = mutableListOf<CarAction>()
+    val history
+        get() = _history.toList()
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        get() = _history
```